docs/MO_DG/prepare_model/FP16_Compression.md

```diff
@@ -2,12 +2,12 @@
 
 @sphinxdirective
 
-Model Optimizer can convert all floating-point weights to the ``FP16`` data type. 
+MO can convert all floating-point weights to the ``FP16`` data type. 
```

Comment:
Please refer to `mo` only in case when it is specific for CLI only. FP16 compression is not specific, it is available in both `convert_model` and `mo`.
```suggestion
Optionally all relevant floating-point weights can be compressed the ``FP16`` data type during the model conversion.
```